hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_derivative.py

```diff
@@ -590,26 +672,27 @@ async def _execute_cancel(self, trading_pair: str, order_id: str) -> str:
             )
 
             response_code = response["ret_code"]
-            if response_code != 0:
-                if response_code == CONSTANTS.ORDER_NOT_EXISTS_ERROR_CODE:
+
+            if response_code != CONSTANTS.RET_CODE_OK:
+                if response_code == CONSTANTS.RET_CODE_ORDER_NOT_EXISTS:
                     self.logger().warning(
-                        f"Failed to cancel order {order_id}:"
+                        f"Failed to cancel order {client_order_id}:"
                         f" order not found ({response_code} - {response['ret_msg']})")
-                    self.stop_tracking_order(order_id)
+                    self.stop_tracking_order(client_order_id)
                 else:
                     raise IOError(f"Bybit Perpetual encountered a problem cancelling the order"
                                   f" ({response['ret_code']} - {response['ret_msg']})")
```

Comment:
![image](https://user-images.githubusercontent.com/7762229/161532208-4a620a66-b836-4708-8058-bf0979896b6f.png)

Due to the previous condition there's a possibility that the order still may exist on the exchange. Therefore the failure is related to connection issues and it can and should be retried by strategies in the next tick.